ui/job-view/headerbars/HelpMenu.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+export default function HelpMenu() {
+  return (
+    <span id="help-menu" className="dropdown">
+      <button
+        id="helpLabel"
+        title="Treeherder help"
+        data-toggle="dropdown"
+        className="btn btn-view-nav nav-help-btn dropdown-toggle"
+      >
+        <span className="fa fa-question-circle lightgray nav-help-icon" />
+      </button>
+      <ul
+        className="dropdown-menu nav-dropdown-menu-right icon-menu"
+        role="menu"
+        aria-labelledby="helpLabel"
+      >
+        <li>
```

Comment:
Since these list items receive the same type of information (and things like target, rel and className are repeated), wouldn't it eliminate some code to put the different attributes in an object and map an <li>?

Reply:
Yeah, you're totally right.  I'll do that.  :)